assets/base.css

```diff
@@ -2754,8 +2755,10 @@ details[open] > .header__menu-item .icon-caret {
   padding: 0.8rem 2.4rem;
 }
 
-.header__submenu .header__submenu .header__menu-item {
-  padding-left: 3.4rem;
+.header__submenu .header__submenu .header__menu-item:hover {
+  font-size:1.5rem;
```

Comment:
I agree the font-size does look cheesy, especially since the hover affect on the L1 only underlines. I went ahead and removed the font-size on the hover effect.

I didn't realize the L2 were not getting a hover effect unless they had drop down items! That was a good catch! I added a hover effect on those as well to keep the feedback the same for all of the links :)